Dockerfile.diskmaker

```diff
@@ -3,10 +3,14 @@ WORKDIR /go/src/github.com/openshift/local-storage-operator
 COPY . .
 RUN make build-diskmaker
 
-FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13
```

Comment:
Accidental change. Was experimenting with what binaries are available in builder, but now we're copying the necessary binaries, will revert in new commit.

All changes new changes will be pushed as new commits and rebased before merging.

Reply:
Actually, can't figure out how to get `e2fsprogs` for mkfs.ext2 onto `registry.svc.ci.openshift.org/openshift/origin-v4.0:base` it is on `registry.svc.ci.openshift.org/openshift/release:golang-1.13` though

Reply:
This is what we did in static-provisioner fork - https://github.com/openshift/sig-storage-local-static-provisioner/blob/master/Dockerfile.rhel7 . We should not try and copy those binaries but install them

Reply:
Hey, I tried that, but it's asking for a subscription to be activated when I tried that in `Dockerfile.diskmaker` it'll probably work in `Dockerfile.diskmaker.rhel7` since that's the same image. I'm not clear about the purpose of the two Dockerfiles, is the rhel one what we actually use downstream?  Is it ok if we just use the builder image for `Dockerfile.diskmaker`?

Reply:
(builder is ~1GiB while origin base is ~300MiB)

Reply:
`Dockerfile.diskmaker` is used by us the developers. We can pretty much use anything as base here. But `Dockerfile.diskmaker.rhel7` is used by ART team and should not require subscription when it is installed.

Reply:
ack, will go with this approach then